src/article/components/AuthorsListComponent.js

```diff
@@ -28,8 +28,15 @@ export default class AuthorsListComponent extends CustomSurface {
   }
 
   render($$) {
-    const el = $$('div').addClass('sc-authors-list');
-    el.append(this._renderAuthors($$));
+    const el = $$('div').addClass('sc-authors-list-container');
```

Comment:
I think this should be a button component rather than an Icon (Which I think I've got wrong myself in a few places!) A good example to look at would be the BoxComponent, which has a button. If you use that object, then when you 'hover' over with the mouse the cursor will properly change.

Reply:
done